scipy/sparse/dok.py

```diff
@@ -414,6 +411,9 @@ def __itruediv__(self, other):
         else:
             return NotImplemented
 
+    def __reduce__(self):
+        return dict.__reduce__(self)
```

Comment:
I tried that but I didn't manage to do it, it was never entering ``__setstate__``. I think because ``dict`` is subclassed and that forces the use of ``__reduce__``? The current solution is just calling ``dict.__reduce__`` reduce instead of ``spmatrix.__reduce__`` in py3 and I assume that's what was already happening in py2. If you have an idea about other implementation let me know so I can give it a shot.